app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1207,23 +1261,28 @@ public void onLoadingFailed(final String imageUri, final View view,
     @Override
     public void onLoadingComplete(final String imageUri, final View view,
                                   final Bitmap loadedImage) {
-        final float width = Math.min(
+        // scale the notification thumbnail
+        final float notificationThumbnailWidth = Math.min(
                 context.getResources().getDimension(R.dimen.player_notification_thumbnail_width),
                 loadedImage.getWidth());
+        currentThumbnail = Bitmap.createScaledBitmap(
+                loadedImage,
+                (int) notificationThumbnailWidth,
+                (int) (loadedImage.getHeight()
+                        / (loadedImage.getWidth() / notificationThumbnailWidth)),
+                true);
 
         if (DEBUG) {
             Log.d(TAG, "Thumbnail - onLoadingComplete() called with: "
                     + "imageUri = [" + imageUri + "], view = [" + view + "], "
                     + "loadedImage = [" + loadedImage + "], "
                     + loadedImage.getWidth() + "x" + loadedImage.getHeight()
-                    + ", scaled width = " + width);
+                    + ", scaled notification width = " + notificationThumbnailWidth);
         }
 
-        currentThumbnail = Bitmap.createScaledBitmap(loadedImage,
-                (int) width,
-                (int) (loadedImage.getHeight() / (loadedImage.getWidth() / width)), true);
-        binding.endScreen.setImageBitmap(loadedImage);
         NotificationUtil.getInstance().createNotificationIfNeededAndUpdate(this, false);
+
+        updateEndScreenThumbnail();
```

Comment:
Seeing as the method twice is used twice, it definetly should be javadoc instead of inline comments

Reply:
Done